### PR TITLE
py3.10 compatible, and refactor integration test's k8s_model fixture

### PR DESCRIPTION
**Pull request by addyess**

### CODE REVIEW

File: jobs/integration/conftest.py [hunk 1]
```diff
@@ -106,7 +105,8 @@ class Tools:
 
     def __init__(self, request):
         self._request = request
-        self.requests = aioify(obj=requests)
+        self.requests = requests
+        self.requests.get = asyncify(requests.get)
```
Comment (addyess) [inline jobs/integration/conftest.py:109]: helps remove `aioify` dep from the repo -- 3 less pip packages to install!

File: jobs/integration/conftest.py [hunk 2]
```diff
@@ -193,30 +193,50 @@ async def model(request, tools):
 
 
 @pytest.fixture(scope="module")
-async def k8s_model(model, tools):
+async def k8s_cloud(model, tools):
```
Comment (addyess) [inline jobs/integration/conftest.py:196]: split `add-k8s` and `remove-cloud` into a different fixture `k8s_cloud`

File: requirements.txt [hunk 3]
```diff
@@ -375,7 +369,8 @@ wcwidth==0.2.5
 websocket-client==1.2.3
     # via kubernetes
 websockets==7.0 ; python_version < "3.9"
-websockets==8.1 ; python_version >= "3.9"
+websockets==8.1 ; python_version == "3.9"
+websockets==10.3 ; python_version >= "3.10"
```
Comment (addyess) [inline requirements.txt:373]: did `pip-compile`  this is getting unruly

**Review by kwmonroe — APPROVED** [review]
Nice!

**Pull request merged**